setup.py

```diff
@@ -80,7 +80,7 @@
         # primary requirements for unit and property models
         # maintainers: switch to SPECIAL_DEPENDENCIES_FOR_RELEASE when cutting a release of watertap
         *SPECIAL_DEPENDENCIES_FOR_PRERELEASE,
-        "pyomo>=6.2,<6.6",  # (also needed for units in electrolyte database (edb))
+        "pyomo>=6.6.1",  # (also needed for units in electrolyte database (edb))
```

Comment:
```suggestion
        "pyomo<=6.6.1",  # (also needed for units in electrolyte database (edb))
```
Any reason why this shouldn't be as suggested? Would tests fail using a version of pyomo below 6.6.1?

Reply:
Yes, Pyomo 6.6.0 has known issues that affect us, namely Pyomo/pyomo#2844 and IDAES/idaes-pse#1191. Both have been resolved in Pyomo 6.6.1.